File: tuts/001-lightsail-gs/cpp/getting_started_scenario.cpp
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

#include <aws/core/Aws.h>
#include <aws/lightsail/LightsailClient.h>
#include "lightsail_wrapper.h"
#include <iostream>

int main(int argc, char *argv[]) {
    Aws::SDKOptions options;
    Aws::InitAPI(options);
    {
        Aws::Lightsail::LightsailClient client;
        std::cout << "Running Lightsail getting started scenario..." << std::endl;
        // TODO: setup, interact, teardown
    }
    Aws::ShutdownAPI(options);
    return 0;
}

File: tuts/001-lightsail-gs/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\Lightsail\LightsailClient;

$client = new LightsailClient(['region' => 'us-east-1']);
echo "Running Lightsail getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";
